datahub-frontend/conf/user.props

```diff
@@ -1 +1,2 @@
-datahub:datahub
\ No newline at end of file
+datahub:datahub
+user:user
```

Comment:
Let's remove this

Reply:
I see this is for tests. My only concern would be that.... people are going to get this new user and not going to even know it. It could make things less secure

Reply:
Do you know of any alternative to create users dynamically in tests? This is needed to test permissions across admin & non-admin users.